app/services/terms_of_use/acceptor.rb

```diff
@@ -42,14 +38,15 @@ def terms_of_use_agreement
       @terms_of_use_agreement ||= user_account.terms_of_use_agreements.new(agreement_version: version)
     end
 
-    def asynchronous_update
+    def update_terms_of_use
       terms_of_use_agreement.accepted!
-      SignUpServiceUpdaterJob.perform_async(attr_package_key)
-    end
+      Rails.logger.info('[TermsOfUse] [Acceptor] attr_package key', { icn:, attr_package_key: })
 
-    def synchronous_update
-      terms_of_use_agreement.accepted!
-      SignUpServiceUpdaterJob.new.perform(attr_package_key)
+      if async
+        SignUpServiceUpdaterJob.perform_async(attr_package_key)
```

Comment:
It looks like `perform_async` has a `sync` parameter, wondering if we should use that?

```
      def perform_async(*args)
        if @opts["sync"] == true
          perform_inline(*args)
        else
          @klass.client_push(@opts.merge("args" => args, "class" => @klass))
        end
      end
```

Reply:
oooo nice find. I also didn't know `perform_inline` was a thing

Reply:
I updated it to set the `sync` option